banner.html

```diff
@@ -0,0 +1,80 @@
+<script>
+  var current_page = document.getElementById('banner').getAttribute('data-current-page');
```

Comment:
It appears that `load` will also execute any scripts contained in the page... Can you rewrite this in Jquery? Something like:
```
$( document ).ready(function() {
  var current_page = $('#banner').data('current-page');
  $('#' + current_page).addClass('usa-current');
  if (current_page == 'home_menu') {
    $('#last_hr').hide();
  }
});
```

Reply:
done